### PR TITLE
Fixes for codeq - QA done

### PR DESCRIPTION
**Pull request by svidhani**
- Swagger integration with resoursePath
- Copyright symbol
- variable name declarations
- data parse

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: jazz_codeq/swagger/swagger.json [hunk 1]
```diff
@@ -385,6 +484,86 @@
                             }
                         }
                     }
+                },
+                "x-amazon-apigateway-integration": {
+                    "uri": "arn:aws:apigateway:us-east-1:lambda:path/2015-03-31/functions/arn:aws:lambda:us-east-1:192006145812:function:gitlab180607-jazztest-jazzapitest3-prod/invocations",
+                    "passthroughBehavior": "when_no_templates",
+                    "httpMethod": "POST",
+                    "credentials": "arn:aws:iam::192006145812:role/gitlab180607_lambda2_basic_execution_1",
```
[the remark this answers is not in the document]
Comment (svidhani) [reply jazz_codeq/swagger/swagger.json]: Fixed

File: jazz_codeq/swagger/swagger.json [hunk 2]
```diff
@@ -385,6 +484,86 @@
                             }
                         }
                     }
+                },
+                "x-amazon-apigateway-integration": {
+                    "uri": "arn:aws:apigateway:us-east-1:lambda:path/2015-03-31/functions/arn:aws:lambda:us-east-1:192006145812:function:gitlab180607-jazztest-jazzapitest3-prod/invocations",
```
[the remark this answers is not in the document]
Comment (svidhani) [reply jazz_codeq/swagger/swagger.json]: Fixed

File: jazz_codeq/swagger/swagger.json [hunk 3]
```diff
@@ -225,6 +225,86 @@
                             }
                         }
                     }
+                },
+                "x-amazon-apigateway-integration": {
+                    "uri": "arn:aws:apigateway:us-east-1:lambda:path/2015-03-31/functions/arn:aws:lambda:us-east-1:192006145812:function:gitlab180607-jazztest-jazzapitest3-prod/invocations",
+                    "passthroughBehavior": "when_no_templates",
+                    "httpMethod": "POST",
+                    "credentials": "arn:aws:iam::192006145812:role/gitlab180607_lambda2_basic_execution_1",
```
[the remark this answers is not in the document]
Comment (svidhani) [reply jazz_codeq/swagger/swagger.json]: Fixed

File: jazz_codeq/swagger/swagger.json [hunk 4]
```diff
@@ -225,6 +225,86 @@
                             }
                         }
                     }
+                },
+                "x-amazon-apigateway-integration": {
+                    "uri": "arn:aws:apigateway:us-east-1:lambda:path/2015-03-31/functions/arn:aws:lambda:us-east-1:192006145812:function:gitlab180607-jazztest-jazzapitest3-prod/invocations",
```
[the remark this answers is not in the document]
Comment (svidhani) [reply jazz_codeq/swagger/swagger.json]: Fixed

File: jazz_codeq/utils.js [hunk 5]
```diff
@@ -151,7 +150,7 @@ function getJazzToken(config) {
 		
 		logger.debug("Getting token for calling Environment API...");
 		request(svcPayload, (error, response, body) => {
-			if (response.statusCode === 200 && body && body.data) {
+			if (response.statusCode === 200 && body) {
```
Comment (devsatishm) [inline jazz_codeq/utils.js]: Why did we remove this check -  && body.data?

Comment (svidhani) [reply jazz_codeq/utils.js]: Its coming as string. so doing body.data gives undefined.

Comment (svidhani) [reply jazz_codeq/utils.js]: Updated

**Pull request merged**